DistributionSystemApi/DistributionSystemApi/stylecop-config.ruleset

```diff
@@ -0,0 +1,88 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Microsoft Managed Recommended Rules" Description="These rules focus on the most critical problems in your code, including potential security holes, application crashes, and other important logic and design errors. It is recommended to include this rule set in any custom rule set you create for your projects." ToolsVersion="16.0">
```

Comment:
it will not work without extension for visual studio or without special nuget package for projects. As for me I dont have this extension...